src/node/mod.rs

```diff
@@ -231,6 +231,27 @@ fn devname(dev: dev_t) -> Option<String> {
     Some(String::from_utf8(dev_name).expect("Returned device name is not valid utf8"))
 }
 
+#[cfg(target_os = "openbsd")]
+fn devname(dev: dev_t) -> Option<String> {
+    use std::os::raw::{c_char, c_int};
+    use std::ffi::{CStr};
+
+    let buf: *mut c_char = unsafe {
```

Comment:
No need for an explicit type when it's defined by the API:

```suggestion
    let buf = unsafe {
```

Reply:
Literally copied this from the non-openbsd version above but I am happy to change it.

Reply:
Feel free to clean that one up too.